lib/common/db-renderable-content.js

```diff
@@ -118,6 +120,11 @@ function dbRenderableContentServiceFactory(
 
     DbRenderableContentService.prototype.render = function render (name, options, scope) {
         return this.get(name, scope).then(function (template) {
+            if (!template) {
```

Comment:
@benbp Yeah - I guess that's true; an empty string is a valid template.  I added this check to catch the case where template is undefined.  Maybe `typeof(template) === 'string'` is a more appropriate check.